apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -1361,16 +1363,16 @@ describe('progressReduxTest', () => {
       server.respond();
 
       const expectedDispatchActions = [
-        'progress/CLEAR_RESULTS',
+        'progress/clearResults',
```

Comment:
is this just a different convention to have these in camelCase instead of all caps?

Reply:
With redux-toolkit, the action names change because rather than explicitly defining constants, they are inferred from the names of the action creator functions ([example](https://github.com/code-dot-org/code-dot-org/pull/52264/files#diff-09cec5783bd5da61e6e7bd042871ad3e2888bb51abd01596e747abc7b88b29b1R124))